.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version:
+          - 1.8
+          - 9
+          - 10
+          - 11
+          - 12
+          - 13
+          - 14
+        # - 15 TODO: Include JDK 15 when we can update JaCoCo to 0.8.7
```

Comment:
What is blocking JaCoCo 0.8.7?

Reply:
https://www.jacoco.org/jacoco/trunk/doc/changes.html

`0.8.7` has yet to be released, although there are snapshots available and it [should be soon™️](https://github.com/jacoco/jacoco/milestone/31). It seemed easiest to just hold off on running on JDK 15 until `0.8.7` is released.